android/src/main/java/com/onegini/mobile/sdk/reactnative/exception/OneginiWrapperErrors.kt

```diff
@@ -8,7 +8,7 @@ enum class OneginiWrapperErrors(val code: String, val message: String) {
     RESOURCE_CALL_ERROR("8005", "Resource call finished with error"),
     AUTHENTICATOR_DOES_NOT_EXIST("8006", "The Fingerprint authenticator does not exist"),
     PROFILE_DOES_NOT_EXIST("8007", "The profileId does not exist"),
-    USER_PROFILE_IS_NULL("8008", "The user profile is null"),
+    USER_PROFILE_IS_NULL("8008", "The user profile is null"), //TODO: does this need to exist, we already have PROFILE_DOES_NOT_EXIST RNP-99
```

Comment:
Indeed it can be removed. If the profile is null we already have the code for not correct parameters

Reply:
This erorr is actually just not descriptive and is actually used as a profileId does not exist error. Because we can not pass a non-null profile in authenticateUser forinstance.
I will replace these with PROFILE_DOES_NOT_EXIST.